src/msg/kava/index.js

```diff
@@ -145,16 +67,63 @@ function newMsgRefundAtomicSwap(sender, swapID) {
 }
 
 /***************************************************
- *                   Incentive
+ *                       CDP
  ***************************************************/
 
-function newMsgClaimReward(sender, collateralType, multiplierName) {
+function newMsgCreateCDP(sender, principal, collateral, collateralType) {
```

Comment:
this diff formatted funny because I organized the file by module type alphabetically. I removed `newMsgClaimReward` since it was replaced by `newMsgClaimUSDXMintingReward` and then just moved the CDP stuff upwards in the file, but made no changes to them